### PR TITLE
Cross Language Tutorial

### PR DESCRIPTION
**Pull request by DavidMansolino**
Tutorial 1: https://www.cyberbotics.com/doc/guide/tutorial-1-your-first-simulation-in-webots-20-minutes?version=feature-cross-language-tutorial
Tutorial 4: https://www.cyberbotics.com/doc/guide/tutorial-4-more-about-controllers-20-minutes?version=feature-cross-language-tutorial&tab=c
Tutorial 6: https://www.cyberbotics.com/doc/guide/tutorial-6-4-wheels-robot?version=feature-cross-language-tutorial

### CODE REVIEW

File: docs/guide/tutorial-1-your-first-simulation-in-webots-20-minutes.md [hunk 1]
```diff
@@ -146,57 +146,177 @@ Then save the simulation.
...
+>Fix any compilation errors if necessary.
+>When Webots proposes to reset or reload the world, choose `Reset` and run the simulation.
+%tab-end
+
+%tab "Java"
+> **Hands on #9**: In the scene tree view, select the `controller` field of the `E-puck` node, then use the field editor at the bottom of the Scene Tree view: press the `Select...` button and then select `e-puck_go_forward` in the list.
+>Once the controller is associated with the robot, save the world.
+>Modify the program by inserting an import statement (`import com.cyberbotics.webots.controller.Motor;`), getting the motor devices (`Motor leftMotor = robot.getMotor("left wheel motor");`), and by applying a motor command (`leftMotor.setPosition(10.0);`):
+>```java
+>#import com.cyberbotics.webots.controller.Robot;
```
Comment (omichel) [inline docs/guide/tutorial-1-your-first-simulation-in-webots-20-minutes.md]:
```suggestion
>import com.cyberbotics.webots.controller.Robot;
```

File: docs/guide/tutorial-1-your-first-simulation-in-webots-20-minutes.md [hunk 2]
```diff
@@ -243,6 +364,125 @@ In order to control the motors of the wheels in speed you need to set the target
...
+>
+>while robot.step(TIME_STEP) != -1:
+>    pass
+>```
+%tab-end
+
+%tab "Java"
+> **Hands on #10**: Modify the controller program as shown below, recompile it and run it:
+>```java
+>#import com.cyberbotics.webots.controller.Robot;
```
Comment (omichel) [inline docs/guide/tutorial-1-your-first-simulation-in-webots-20-minutes.md]:
```suggestion
>import com.cyberbotics.webots.controller.Robot;
```

**Comment by DavidMansolino** [comment]
:+1:

**Comment by omichel** [comment]
:eyes:

File: docs/reference/motor.md [hunk 3]
```diff
@@ -159,7 +159,7 @@ wb_motor_set_velocity(motor, 6.28);  // 1 rotation per second
 ```
 
 `INFINITY` is a C macro corresponding to the IEEE 754 floating point standard.
-It is implemented in the C99 specifications as well as in C++.
+It is implemented in the C99 specifications as well as in C++ (you may have to include `<math.h>`).
```
Comment (omichel) [inline docs/reference/motor.md]: It would be better that `motor.h` already includes `math.h` so that we can remove this comment in parenthesis.

**Comment by DavidMansolino** [comment]
test me

**Comment by DavidMansolino** [comment]
test me

**Comment by DavidMansolino** [comment]
:+1:

**Comment by DavidMansolino** [comment]
:+1:

File: docs/guide/tutorial-4-more-about-controllers-20-minutes.md [hunk 4]
```diff
@@ -174,11 +319,289 @@ Compile your code by selecting the `Build / Build` menu item.
...
+> rightMotor.setVelocity(0.0)
+> ```
+In the main loop, just after the comment `# read sensors outputs`, read the distance sensor values as follows:
+> ```python
+> # read sensors outputs
+> psValues = []
+> for i in range(8):
+>     psValues.append(ps[i].getValue())
+> ```
+In the main loop, just after the comment `% process behavior`, detect if a collision occurs (i.e., the value returned by a distance sensor is bigger than a threshold) as follows:
```
Comment (omichel) [inline docs/guide/tutorial-4-more-about-controllers-20-minutes.md]:
```suggestion
In the main loop, just after the comment `# process behavior`, detect if a collision occurs (i.e., the value returned by a distance sensor is bigger than a threshold) as follows:
```

File: docs/guide/tutorial-6-4-wheels-robot.md [hunk 5]
```diff
@@ -160,32 +160,139 @@ In the previous tutorials, you have learnt how to setup a feedback loop and how
...
+Motor[4] wheels;
+String[] wheelsNames = {"wheel1", "wheel2", "wheel3", "wheel4"};
+for (int i = 0; i < wheelsNames.length; i++)
+  wheels[i] = robot.getMotor(wheelsNames[i]);
 ```
+%tab-end
+
+%tab "MATLAB"
+```matlab
+// initialize motors
```
Comment (omichel) [inline docs/guide/tutorial-6-4-wheels-robot.md]:
```suggestion
% initialize motors
```

File: docs/guide/tutorial-6-4-wheels-robot.md [hunk 6]
```diff
@@ -160,32 +160,139 @@ In the previous tutorials, you have learnt how to setup a feedback loop and how
...
+```java
+double speed = -1.5; // [rad/s]
+wheels[0].setPosition(Double.POSITIVE_INFINIT);
+wheels[0].setVelocity(speed);
+```
+%tab-end
+
+%tab "MATLAB"
+```matlab
+speed = -1.5; // [rad/s]
```
Comment (omichel) [inline docs/guide/tutorial-6-4-wheels-robot.md]:
```suggestion
speed = -1.5; % [rad/s]
```

**Pull request merged**